source/_integrations/mqtt.markdown

```diff
@@ -559,44 +559,233 @@ The following software has built-in support for MQTT discovery:
 
 ### Discovery examples
 
-#### Motion detection (binary sensor)
+#### Creating a single sensor entity (motion detection binary sensor)
 
-A motion detection device which can be represented by a [binary sensor](/integrations/binary_sensor.mqtt/) for your garden would send its configuration as JSON payload to the Configuration topic. After the first message to `config`, then the MQTT messages sent to the state topic will update the state in Home Assistant.
+In the following example, a single entity will be created representing the state of a motion detection sensor. The motion detection sensor has only 2 states, therefore it can be represented by an [MQTT Binary Sensor](/integrations/binary_sensor.mqtt/).
+
+A user, script, or device would send its configuration as JSON payload to the Configuration topic. After the first message to `config`, then the MQTT messages sent to the state topic will update the motion detection sensor state in Home Assistant.
 
 - Configuration topic: `homeassistant/binary_sensor/garden/config`
-- State topic: `homeassistant/binary_sensor/garden/state`
-- Payload:  `{"name": "garden", "device_class": "motion", "state_topic": "homeassistant/binary_sensor/garden/state"}`
+- Configuration payload:  `{"name": "garden", "device_class": "motion", "state_topic": "homeassistant/binary_sensor/garden/state"}`
 - Retain: The -r switch is added to retain the configuration topic in the broker. Without this, the sensor will not be available after Home Assistant restarts.
 
-To create a new sensor manually.
+To create a new sensor manually:
 
 ```bash
 mosquitto_pub -r -h 127.0.0.1 -p 1883 -t "homeassistant/binary_sensor/garden/config" -m '{"name": "garden", "device_class": "motion", "state_topic": "homeassistant/binary_sensor/garden/state"}'
 ```
 
-Update the state.
+Update the state:
 
 ```bash
 mosquitto_pub -h 127.0.0.1 -p 1883 -t "homeassistant/binary_sensor/garden/state" -m ON
 ```
 
-Delete the sensor by sending an empty message.
+Delete the sensor by sending an empty message:
 
  ```bash
 mosquitto_pub -h 127.0.0.1 -p 1883 -t "homeassistant/binary_sensor/garden/config" -m ''
 ```
 
-For more details please refer to the [MQTT testing section](/integrations/mqtt/#testing-your-setup).
+Note: the Configuration topic prefix must be `homeassistant` (or another if you changed the default). However, the State topic prefix does not need to be `homeassistant`. It is common for the State topic prefix to indicate which device or script sent the MQTT message, for example 'my_esp_device', 'rtl_433', or 'my_python_script'.
```

Comment:
This note is not needed as it is well documented already.

Reply:
Perhaps the next example is sufficient to illustrate that the 'homeassistant' prefix is not required to be part of the State topic. Would you agree?

Reply:
I don't like the big changes in this PR. Can you split typo's small corrections. Larger changes should not be all together as this is hard to review. A lot of the stuff you add I don't want at this page or I do not think changes are needed. I'll do a closer review later.